editor/plugins/material_editor_plugin.cpp

```diff
@@ -288,16 +314,37 @@ MaterialEditor::MaterialEditor() {
 	quad_instance = memnew(MeshInstance3D);
 	rotation->add_child(quad_instance);
 
-	sphere_instance->set_transform(Transform3D() * 0.375);
-	box_instance->set_transform(Transform3D() * 0.25);
-	quad_instance->set_transform(Transform3D() * 0.375);
+	floor_instance = memnew(MeshInstance3D);
+	rotation->add_child(floor_instance);
+
+	Transform3D transform = Transform3D();
+	transform.origin = Vector3(0, 0.5, 0);
+	sphere_instance->set_transform(transform);
+	box_instance->set_transform(transform);
+	quad_instance->set_transform(transform);
 
 	sphere_mesh.instantiate();
 	sphere_instance->set_mesh(sphere_mesh);
 	box_mesh.instantiate();
 	box_instance->set_mesh(box_mesh);
 	quad_mesh.instantiate();
 	quad_instance->set_mesh(quad_mesh);
+	floor_mesh.instantiate();
+	floor_mesh->set_size(Size2(5, 5));
+	floor_instance->set_mesh(floor_mesh);
+
+	contents_aabb = sphere_instance->get_transform().xform(sphere_mesh->get_aabb());
+
+	default_floor_material = memnew(StandardMaterial3D);
+	default_floor_material->set_uv1_scale(Vector3(20, 20, 20));
+	default_floor_material->set_texture_filter(StandardMaterial3D::TEXTURE_FILTER_NEAREST);
+	default_floor_material->set_albedo(Color::hex(0x454545ff));
+	floor_mesh->set_material(default_floor_material);
+
+	probe = memnew(ReflectionProbe);
+	probe->set_size(Vector3(5, 1.5, 5));
```

Comment:
```suggestion
	probe->set_size(Vector3(5, 1.5, 5));
	// Disable blending as the probe fully covers the entire preview scene.
	probe->set_blend_distance(0.0);
```

Before | After
-|-
![Before](https://github.com/user-attachments/assets/b11ac0ce-2e88-4e23-a9f5-ec16dc7f62e5) | ![After](https://github.com/user-attachments/assets/3afc7f55-cf6b-4daa-bc34-415e09236e0e)